server/formulas/nutrition.formulas.js

```diff
@@ -0,0 +1,51 @@
+import {
+  CALORIES
+} from './measurement';
+
+const FREQUENCY_MULTIPLIER = {
+  sedentary: 1.2,
+  light: 1.375,
+  moderate: 1.55,
+  active: 1.725
+};
+
+export function getTdee(exerciseFrequency, weight, height, age, sex) {
+  if (!(exerciseFrequency in FREQUENCY_MULTIPLIER)) {
+    throw new Error(`Unrecognized exercise frequency ${exerciseFrequency}.`);
+  }
+
+  let ree;
+
+  if (sex === 'male') {
+    ree = 10 * weight.get('kg') + 6.25 * height.get('cm') - 5 * age + 5;
```

Comment:
I'm actually a little skeptical of these formulas, these seem a little _too_ similar

Reply:
What do you propose we do?

Reply:
alright according to https://healthyeater.com/how-to-calculate-your-macros this is actually correct?